_neural-search-plugin/index.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Neural Search plugin
+nav_order: 1
+has_children: false
+has_toc: false
+---
+
+The OpenSearch Neural Search plugin enables the integration of Machine Learning (ML) language models into their search workloads. Because language models can sometime require a over 100mb of memory, the Neural Search plugin automatically turns your index documents into vectors and ingests your vectorized documents into an OpenSearch index.
```

Comment:
Suggestion: The OpenSearch Neural Search plugin enables the integration of Machine Learning (ML) language models into user's search workloads. During ingestion and search, users can provide text that the neural search plugin will automatically transform into vectors to be used in vector search.